wlc/test_base.py

```diff
@@ -63,9 +63,15 @@ def format_body(body):
             return ""
         body = body.decode()
         body = (
-            body.replace(": ", "=").replace("{", "").replace("}", "").replace('"', "")
+            body.replace(": ", "=")
+            .replace("{", "")
+            .replace("}", "")
+            .replace('"', "")
+            .replace("://", "-::")
```

Comment:
Colon is not allowed on Windows, see https://github.com/WeblateOrg/wlc/actions/runs/555135356.